src/Invoice.php

```diff
@@ -146,6 +200,17 @@ public function setInvoiceDate($invoiceDate)
         return $this;
     }
 
+    public function getDueDate()
```

Comment:
Can you make this work with DateTimeInterface types?

Reply:
Fixed in #142